app/components/scheduled_post_indicator/scheduled_post_indicator.tsx

```diff
@@ -41,6 +33,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
         link: {
             color: theme.linkColor,
             fontSize: 14,
+            fontWeight: '600',
```

Comment:
We should be using `typography` for this.

Reply:
Done.